R/bs-theme.R

```diff
@@ -128,11 +128,15 @@ bs_theme <- function(version = version_default(), preset = NULL, ...,
   preset <- resolve_bs_preset(preset, bootswatch, version = version)
 
   bundle <- bs_bundle(
-    bs_theme_init(version, subclass = preset$class),
+    bs_theme_init(version),
     bootstrap_bundle(version),
     bs_preset_bundle(preset)
   )
 
+  if (!is.null(preset$theme_class)) {
+    bundle <- add_class(bundle, preset$theme_class)
+  }
```

Comment:
I still doesn't feel quite right that `$theme_class` exists. Could we get rid of it, and do something like this?

```suggestion
  if (!is.null(preset$type)) {
    bundle <- add_class(bundle, theme_preset_class())
  }
```

Reply:
I don't mind, I made the change to a constant in 3c44cc32d2efe6c70cb1998964039bbc2b827d2d. 

The reason I didn't originally was because it felt like `preset$type` being NULL was an indirect signal of whether this class should be added. I included `preset$theme_class` so that if we changed that assumption later we'd only need to reason about the `preset` object.

But the problem is that we have to remove preset the preset class as part of `bs_theme_update()` ... so I'm on board with constant as a the signal that we're doing something special.